### PR TITLE
v4.0.0-alpha3

### PR DESCRIPTION
**Pull request by danbryan**

### CODE REVIEW

File: testnet/grand-1/chain-upgrades/v4.0.0-alpha3/README.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Upgrade to v4.0.0-alpha3
+
+## Overview
+
+We are excited to announce the next software upgrade for the Noble `grand-1` testnet chain. This upgrade will transition the network to `v4.0.0-alpha3`. This upgrade adds TBD.
+
+**Key Details:**
+- **Target Block:** [`2682700`](https://testnet.mintscan.io/noble-testnet/blocks/2682700)
```
Comment (johnletey) [inline testnet/grand-1/chain-upgrades/v4.0.0-alpha3/README.md:8]:
```suggestion
- **Target Block:** [`2772300`](https://testnet.mintscan.io/noble-testnet/blocks/2772300)
```

File: testnet/grand-1/chain-upgrades/v4.0.0-alpha3/README.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## Step-by-Step Upgrade Instructions
+
+1. **Set Halt Height:** All validators must set the halt height to `2682700`. There is no on chain instruction to halt.
+2. **Use Pre-Built Binaries:** Choose the appropriate binary for your system:
+   - [AMD](./nobled_linux-amd64)
+   - [ARM](./nobled_linux-arm64)
+
+## Update
+On 8/25/23 we identified that the AMD binary was wrong. We have updated the binary and the checksums below. To resolve this issue we would like to do another halt-height upgrade at block [2700000](https://testnet.mintscan.io/noble-testnet/blocks/2700000)
```
Comment (johnletey) [inline testnet/grand-1/chain-upgrades/v4.0.0-alpha3/README.md:19]:
```suggestion
```

File: testnet/grand-1/chain-upgrades/v4.0.0-alpha3/README.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
+# Upgrade to v4.0.0-alpha3
+
+## Overview
+
+We are excited to announce the next software upgrade for the Noble `grand-1` testnet chain. This upgrade will transition the network to `v4.0.0-alpha3`. This upgrade adds TBD.
+
+**Key Details:**
+- **Target Block:** [`2682700`](https://testnet.mintscan.io/noble-testnet/blocks/2682700)
+- **Estimated Date & Time:** Thursday, August 24th, around 11am EST. *(Please note that this is an approximation. Due to varying block times, it's crucial to monitor the chain for the exact upgrade time.)*
```
Comment (johnletey) [inline testnet/grand-1/chain-upgrades/v4.0.0-alpha3/README.md:9]:
```suggestion
- **Estimated Date & Time:** Wednesday, August 30th, around 11am EST. *(Please note that this is an approximation. Due to varying block times, it's crucial to monitor the chain for the exact upgrade time.)*
```

File: testnet/grand-1/chain-upgrades/v4.0.0-alpha3/README.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+We are excited to announce the next software upgrade for the Noble `grand-1` testnet chain. This upgrade will transition the network to `v4.0.0-alpha3`. This upgrade adds TBD.
+
+**Key Details:**
+- **Target Block:** [`2772300`](https://testnet.mintscan.io/noble-testnet/blocks/2772300)
+- **Estimated Date & Time:** Wednesday, August 30th, around 11am EST. *(Please note that this is an approximation. Due to varying block times, it's crucial to monitor the chain for the exact upgrade time.)*
+
+## Step-by-Step Upgrade Instructions
+
+1. **Set Halt Height:** All validators must set the halt height to `2682700`. There is no on chain instruction to halt.
```
Comment (boojamya) [inline testnet/grand-1/chain-upgrades/v4.0.0-alpha3/README.md:13]: I don't think we need to set the halt height. 
We have an upgrade handler/name `argon2`

File: testnet/grand-1/chain-upgrades/v4.0.0-alpha3/README.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+   - [AMD](./nobled_linux-amd64)
+   - [ARM](./nobled_linux-arm64)
+
+
+### Using Docker?
+
+You can grab the binaries directly from the latest image with:
+
+```shell
+docker pull ghcr.io/strangelove-ventures/heighliner/noble:v4.0.0-alpha3
```
Comment (boojamya) [inline testnet/grand-1/chain-upgrades/v4.0.0-alpha3/README.md:24]: Just a note, but these images are pushed to both `heighliner` and the noble repo. So we can have them pull from either one. 
Maybe `noble` is more intuitive? 
https://github.com/strangelove-ventures/noble/pkgs/container/noble/123165979?tag=v4.0.0-alpha3

**Review by agouin — APPROVED** [review]

**Pull request merged**